### PR TITLE
Updated srs

### PR DESCRIPTION
**Pull request by MadsHeilmann**

### CODE REVIEW

File: src/problem-solution/srs.tex [hunk 1]
```diff
@@ -1,5 +1,35 @@
 % textidote: ignore begin
-\section{Software Requirements Specification}
+\section{Software Requirements Specification}\label{sec:software-requirements-specification}
 % textidote: ignore end
 
-The outline of the requirements of the program is divided into two categories; hard requirements and soft requirements.
+The outline of the requirements of the program is divided into two categories; hard requirements labeled ''need to
+haves'' and soft requirements labeled ''nice to haves''.
+''Need to haves'' include all functions of the program which is expected to be included and implemented.
+In contrast, the ''Nice to haves'' include functions which are out of scope of this report, but would improve the
```
Comment (Danar435) [inline src/problem-solution/srs.tex]: '' før nice/need to haves skal være ``

**Review by Danar435 — APPROVED** [review]

**Review by TexWO — APPROVED** [review]

File: src/problem-analysis/emissions.bib [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (audio-engineer) [inline src/problem-analysis/emissions.bib]: @MadsHeilmann Unnecessary empty line

File: src/problem-solution/srs.tex [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
 % textidote: ignore end
 
-The outline of the requirements of the program is divided into two categories; hard requirements and soft requirements.
+% textidote: ignore begin
+The outline of the requirements of the program is divided into two categories; hard requirements labeled ``need to
+have's'' and soft requirements labeled ``nice to have's''.
+``Need to have's'' include all functions of the program which is expected to be included and implemented.
+In contrast, the ``nice to have's'' include functions which are out of scope of this report, but would improve the
+program if implemented.
+% textidote: ignore end
```
Comment (audio-engineer) [inline src/problem-solution/srs.tex:11]: @MadsHeilmann Why are you ignoring 5 lines? Have you tried ignoring one line only?
```suggestion
% textidote: ignore begin
In contrast, the ``nice to have's'' include functions which are out of scope of this report, but would improve the program if implemented.
% textidote: ignore end
```

**Review by audio-engineer — CHANGES_REQUESTED** [review]

File: src/problem-solution/srs.tex [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
 
-The outline of the requirements of the program is divided into two categories; hard requirements and soft requirements.
+% textidote: ignore begin
+The outline of the requirements of the program is divided into two categories; hard requirements labeled ``need to
+have's'' and soft requirements labeled ``nice to have's''.
+``Need to have's'' include all functions of the program which is expected to be included and implemented.
+In contrast, the ``nice to have's'' include functions which are out of scope of this report, but would improve the
+program if implemented.
+% textidote: ignore begin
+In contrast, the ``nice to have's'' include functions which are out of scope of this report, but would improve the program if implemented.
```
Comment (Danar435) [inline src/problem-solution/srs.tex]: Line repeated twice, why not delete line 11-13?

**Review by Danar435 — CHANGES_REQUESTED** [review]

**Review by Danar435 — APPROVED** [review]

File: src/problem-solution/srs.tex (hunk 3 above)
Comment (MadsHeilmann) [reply src/problem-solution/srs.tex:11]: Yes - I have now made it cover fewer lines

**Review by audio-engineer — APPROVED** [review]

**Comment by audio-engineer** [comment]
@MadsHeilmann I see that you have some merge conflicts. Do you need my help in resolving them or will you do it yourself?

**Pull request merged**